src/components/combobox/combobox.stories.ts

```diff
@@ -297,6 +297,20 @@ export const WithPlaceHolderIcon = (): string => html` <calcite-combobox
   <calcite-combobox-item value="2" text-label="Folder 2" icon="folder"></calcite-combobox-item>
 </calcite-combobox>`;
 
+export const WithPlaceHolderTextAndNoIcon = (): string => html` <calcite-combobox
+  id="labelFour"
```

Comment:
Make sure to only specify the attributes needed for the test. We want tests to be as simple as possible.